hostscripts/rpm-packaging/createproject.py

```diff
@@ -82,10 +82,10 @@ def upload_meta(project, build_repository, linkproject):
                   'projectlink': projectlink,
                   'build_repository': build_repository})
 
-    with tempfile.NamedTemporaryFile(delete=False) as meta:
+    with tempfile.NamedTemporaryFile() as meta:
         meta.write(templ)
+        meta.file.flush()
```

Comment:
Because is a file-like. This means that `.flush()` and other methods like `.seek()` are not implemented at NTF level.